packages/taiko-client/prover/init.go

```diff
@@ -160,22 +160,117 @@ func (p *Prover) initProofSubmitters(
 			p.proofSubmittersOntake = append(p.proofSubmittersOntake, submitter)
 		}
 	}
+	return p.initPacayaProofSubmitter(txBuilder)
+}
+
+func (p *Prover) initPacayaProofSubmitter(txBuilder *transaction.ProveBlockTxBuilder) error {
+	// Init verifiers for pacaya
```

Comment:
```suggestion
	// Init verifiers for Pacaya
```

plz use `Pacaya` in all comments instead of `pacaya`.

Reply:
Sounds good